dir.props

```diff
@@ -56,22 +56,15 @@
   <!-- Provides package dependency version properties and verification/auto-upgrade configuration -->
   <Import Project="$(MSBuildThisFileDirectory)dependencies.props" />
 
-  <ItemGroup>
-    <DotnetSourceList Include="$(DotNetSourceBuildRestoreSources)" />
-  </ItemGroup>
-
   <!-- list of nuget package sources passed to nuget restore -->
-  <ItemGroup Condition="'$(DotNetBuildOffline)' != 'true'">
-    <!-- Example to consume local CoreCLR package:
-         /p:OverridePackageSource=C:\coreclr\bin\Product\Windows_NT.x64.Debug\.nuget\pkg
-    -->
-    <DotnetSourceList Include="$(OverridePackageSource)" />
```

Comment:
I learned this the hard way, but `OverridePackageSource` is used in our scripts.  So we still need to respect that property.

https://github.com/dotnet/corefx/blob/b951d70e7ed15e4194b7e306cc87751b6cf76c24/cross/arm32_ci_script.sh#L237

Reply:
Good call we should keep that.